metricbeat/tests/system/test_galera.py

```diff
@@ -0,0 +1,43 @@
+import os
+import metricbeat
+import unittest
+from nose.plugins.attrib import attr
+
+GALERA_FIELDS = metricbeat.COMMON_FIELDS + ["galera"]
+
+GALERA_STATUS_FIELDS = ["local.state", "evs.state", "cluster.size", "cluster.status", "connected", "ready"]
+
+
+class Test(metricbeat.BaseTest):
+
+    COMPOSE_SERVICES = ['galera']
```

Comment:
This requires a `galera` service in the `docker-compose.yml` file.

Reply:
I did overlooked this indeed. Since there is no separated service for galera this is no longer needed. 

As I mentioned earlier, a docker test for this would be very difficult, since a complete cluster (>= 3 nodes) has to be deployed for this in a docker-container.

Reply:
Ok, this can be covered in a follow up PR.